gui/default/index.html

```diff
@@ -222,7 +222,9 @@ <h3 translate>Folders</h3>
               <div class="panel-progress" ng-show="folderStatus(folder) == 'syncing'" ng-attr-style="width: {{syncPercentage(folder.id)}}%"></div>
               <div class="panel-progress" ng-show="folderStatus(folder) == 'scanning' && scanProgress[folder.id] != undefined" ng-attr-style="width: {{scanPercentage(folder.id)}}%"></div>
               <h3 class="panel-title">
-                <span class="fa hidden-xs fa-fw" ng-class="[folder.readOnly ? 'fa-lock' : 'fa-folder']"></span>{{folder.id}}
+                <span class="fa hidden-xs fa-fw" ng-class="[folder.readOnly ? 'fa-lock' : 'fa-folder']"></span>
+                <span                                  ng-show="folder.label.length == 0" >{{folder.id   }}</span>
+                <span title="folder id: {{folder.id}}" ng-show="folder.label != undefined">{{folder.label}}</span>
```

Comment:
I'm confused about the one thing checking folder.label.length and the other checking it's undefinedness. I also dislike the indentation, we don't do that anywhere else. :) (And we should not put untranslated strings into the title, so maybe just stick the folder ID there without comment for simplicity.)